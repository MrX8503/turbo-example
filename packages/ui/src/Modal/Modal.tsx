import type { ReactNode } from 'react';

export interface ModalProps {
    children?: ReactNode;
}

export default function Modal ({ children }: ModalProps) {
    return (
        <>
            <h1>Modal Component</h1>
            {children}
        </>
    );
}
