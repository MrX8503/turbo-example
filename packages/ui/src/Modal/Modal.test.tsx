import renderer from 'react-test-renderer';

import Modal from './Modal';

describe('Modal', () => {
    it('snapshot is unchanged', () => {
        const toRender = (
            <Modal>
                Hello World
            </Modal>
        );

        const tree = renderer.create(toRender).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
