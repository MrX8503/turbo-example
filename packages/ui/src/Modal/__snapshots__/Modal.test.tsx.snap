// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal snapshot is unchanged 1`] = `
[
  <h1>
    Modal Component
  </h1>,
  "Hello World",
]
`;
